.github/workflows/conversion_script.yml

```diff
@@ -0,0 +1,63 @@
+name: Generate Conversion Script
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*_diff.txt'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/all_ontology.json.gz'
+
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-conversion-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.11
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install -r migration_assistant/requirements.txt
+      - name: Run Conversion Script Generator
+        run: |
+          make generate-conversion-script
+      - name: Get Version
+        id: get_version
+        run: |
+          echo "old_version=$(make show-current-version)" >> $GITHUB_ENV
+      - name: setup git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Bump Version
```

Comment:
We need to change the conditions that trigger this workflow and the publish-to-PyPI workflow. bumpversion is set up to commit, and both that workflow and this workflow call it + have the same triggers. So we'll end up bumping the version twice across both workflows.

Reply:
easiest route is likely to have a bump version workflow triggered by the ontology reference file updates, which in turn triggers both this workflow + publish-minor-to-pypi

Reply:
I move the GHA into publish-to-pypy.yml and running it after the package is published.